utils/src/fuse.rs

```diff
@@ -252,12 +252,25 @@ fn fuse_kern_mount(
     subtype: &str,
     flags: MsFlags,
 ) -> io::Result<File> {
-    let file = OpenOptions::new()
+    let file = match OpenOptions::new()
         .create(false)
         .read(true)
         .write(true)
-        .open(FUSE_DEVICE)?;
-    let meta = mountpoint.metadata()?;
+        .open(FUSE_DEVICE)
+    {
+        Ok(v) => v,
+        Err(e) => {
+            error!("FUSE failed to open. {}", e);
```

Comment:
How about just using error mapping. For example:
```
let file = OpenOptions::new()
        .create(false)
        .read(true)
        .write(true)
        .open(FUSE_DEVICE).map_err(|e| {error!("bla-bla");e})?
```
That is neater and clean

Reply:
yes, I agree

Reply:
Yes, I agree